yarn.lock

```diff
@@ -5999,12 +5511,7 @@ minimist-options@4.1.0:
     is-plain-obj "^1.1.0"
     kind-of "^6.0.3"
 
-minimist@^1.1.1, minimist@^1.2.0, minimist@^1.2.5:
```

Comment:
Can we use yarn why to verify this ?

Reply:
Similar problem: A few years back we looked into [yarn-deduplicate](https://www.npmjs.com/package/yarn-deduplicate) as a post-processor to ensure duplicate versions were removed and . Would love if the UI Core team looked into hooking into this (or a similar lib) for cleaning up our lock files.

Reply:
```sh
info Reasons this module exists
   - "_project_#@okta#eslint-plugin-okta#@babel#core#json5" depends on it
   - Hoisted from "_project_#@okta#eslint-plugin-okta#@babel#core#json5#minimist"
   - Hoisted from "_project_#lerna#@lerna#create#@lerna#child-process#strong-log-transformer#minimist"
   - Hoisted from "_project_#npm-registry-fetch#make-fetch-happen#cacache#tar#mkdirp#minimist"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#conventional-changelog-writer#handlebars#minimist"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#run-lifecycle#npm-lifecycle#node-gyp#mkdirp#minimist"
```

From what I saw in `yarn.lock`, it's used by handlebars, json5, strong-log-transformer, and mkdirp. The latter has 3 versions installed. One requires 1.2.5 and another 1.2.6. The last one (newest) removed this dependency.

All of them had carrots `^` on the version, so I dunno how we wound up with 2 separate installs in `yarn.lock. The same issue is there with `mkdirp` where there are 2 versions at `0.5.x`.

I looked at the history of `yarn.lock` and noticed a second was added in this commit:
![image](https://user-images.githubusercontent.com/97464104/176309549-5bd6ada8-b31f-44d6-8978-101a7bc84405.png)

It's possible the way we need this upgrade ended up being destructive in some way. If I remove the `yarn.lock` and reinstall, we should only have a single entry.